index.html

```diff
@@ -1127,47 +1127,126 @@ <h2>Relative DID URLs</h2>
   <section>
     <h1>Data Model</h1>
     <p>
-This specification defines an abstract data model for <a>DID documents</a>,
-independent of any specific representation. This section provides a
-high-level description of the data model, a set of requirements for
-representations, and a set of requirements for extensibility.
+This specification defines an abstract data model for <a>DID documents</a> that
+is capable of being serialized into multiple concrete representations. This
+section provides a high-level description of the data model, how different types
+of properties are expressed in the data model, and instructions for extending
+the data model.
     </p>
-    <section>
-      <h2>Definition</h2>
-      <p>
+    <p>
 A <a>DID document</a> consists of a <a data-cite="INFRA#maps">map</a> of <a
-data-cite="INFRA#map-entry">properties</a>, which are name-value pairs (ie. a
-property name, and a property value). The definitions of each of these
-properties are specified in section <a href="#core-properties"></a>. Specific
-representations are defined in section <a href="#core-representations"></a>.
-      </p>
-    </section>
-    <section>
-      <h2>Representations</h2>
-      <p>
-Following are the requirements for representations.
-      </p>
-      <ol>
-        <li>
-A representation MUST define an unambiguous encoding and decoding of all
-property names and their associated values as defined in this specification.
-This means anything you can represent in the <a>DID document</a> data model
-can be represented in any compliant representation.
-        </li>
-        <li>
-The representation MUST be associated with an IANA-registered MIME type.
-        </li>
-        <li>
-The representation MUST define fragment processing rules for its MIME type that
-are conformant with the fragment processing rules defined in section
-<a href="#fragment"></a> of this specification.
-        </li>
-      </ol>
-        <p>
-The core representations are specified in section
-<a href="#core-representations"></a>.
-        </p>
-    </section>
+data-cite="INFRA#map-entry">properties</a>, where each property consists of
+a property name/property value pair. The data model contains at least two
+different classes of properties. The first class of properties are called
+core properties, and are specified in section <a href="#core-properties"></a>.
+The second class of properties are called representation properties, and
+are specified in section <a href="#representations"></a>.
```

Comment:
I like the two buckets. I'm not sure if we want to bikeshed the names here, or whether we should worry about that after the primary changes in the spec (i.e., after this PR is merged).

Reply:
I expect we'll bike shed the names in another PR. This PR assumes that we'll do that later.